### DISCUSSION TITLE
at checkout, country select field fails to display default value

### DISCUSSION THREAD

**Discussion by delfina-rossi** [breezefront/community#331] [1. Help]
I can see some differences between address-validation.js from breeze and magento original, not sure if that's a clue.

I'm using a standard breeze checkout and if I go to checkout unlogged and with clear site data, the country field fails to setup the default value which is US. If I'm logged in, or if I've selected it before it does appear by default.

![image](https://github.com/user-attachments/assets/e59f9811-8368-4a69-92d2-83a87d3cfd36)

If I switch to Luma it works fine in all scenarios.

**Comment by vovayatsyuk** [comment]
That's weird because we do not change checkout page. At checkout standard Magento JS components are used only.
